### ISSUE TITLE
Installation failed on Windows 10 (WSL)

### ISSUE THREAD

**Issue by snizemic** [welaika/wordmove#412]
Maybe I am doing something wrong. I tried install Wordmove on WSL but without success. I did everything step by step according to guidelines [https://github.com/welaika/wordmove/wiki/How-to-install-Wordmove-on-Windows-10-with-Bash](https://github.com/welaika/wordmove/wiki/How-to-install-Wordmove-on-Windows-10-with-Bash) when I tried `gem install wordmove` (also sudo) I obtain `photocopier requires Ruby version ~> 2.4.` Update Ruby to 2.4 is not working.

**Comment by pioneerskies** [comment]
Said I have not a Win install where I could try, but just to clarify: what is your ruby's version at the moment?

**Comment by snizemic** [comment]
Yes, I have
```
Distributor ID: Ubuntu
Description: Ubuntu 16.04.2 LTS
Release: 16.04
Codename: xenial
```
```ruby 2.3.4p301 (2017-03-30 revision 58214 [x86_64-linux-gnu]```

**Comment by pioneerskies** [comment]
So as far as I can understand, you have not succeeded in upgrading ruby. In that case I'd use rvm or [rbenv](https://github.com/rbenv/rbenv) in order to use an arbitrary ruby version on your system, ignoring the default versione shipped by your OS

**Comment by snizemic** [comment]
I have to correct my previous post. I ran `apt update` and `apt upgrade` then I removed ruby 2.3 and installed ruby 2.4 and now I am able to call wordmove commands, e.g. `wordmove --help` etc., and I get response without errors now. So I have to try to move some installation of WordPress.
I presume the mistake was really in ruby version. Although I updated to version 2.4 there still was version 2.3 in the WSL and it maybe made some conflicts. I am not a linux guru :-)
